rebar.config

```diff
@@ -52,97 +52,74 @@
     ]}
 ]}.
 
+{relx, [
+        {release, {'miner', "0.1.0"},
+         [miner, observer, runtime_tools, tools,wx, recon]},
+        {sys_config, "./config/dev.config"},
+        {overlay,
+         [
+          {copy, "scripts/extensions/genesis", "bin/extensions/genesis"},
+          {copy, "scripts/extensions/hbbft", "bin/extensions/hbbft"},
+          {copy, "scripts/extensions/info", "bin/extensions/info"},
+          {copy, "scripts/extensions/dkg", "bin/extensions/dkg"},
+          {copy, "scripts/extensions/authorize", "bin/extensions/authorize"},
+          {copy, "scripts/extensions/print_keys", "bin/extensions/print_keys"},
+          {copy, "./_build/default/lib/blockchain/scripts/extensions/peer", "bin/extensions/peer"},
+          {copy, "./_build/default/lib/blockchain/scripts/extensions/ledger", "bin/extensions/ledger"},
+          {copy, "./_build/default/lib/blockchain/scripts/extensions/trace", "bin/extensions/trace"},
+          {copy, "./_build/default/lib/blockchain/scripts/extensions/txn", "bin/extensions/txn"},
+          {copy, "./_build/default/lib/blockchain/scripts/extensions/repair", "bin/extensions/repair"},
+          {template, "config/vm.args", "{{output_dir}}/releases/{{release_version}}/vm.args"}
+         ]},
+        {extended_start_script_hooks,
+         [
+          {post_start,
+           [
+            {wait_for_process, blockchain_worker}
+           ]}
+         ]},
+        {generate_start_script, true},
+        {extended_start_script, true},
+        {extended_start_script_extensions,
+         [
+          {genesis, "extensions/genesis"},
+          {hbbft, "extensions/hbbft"},
+          {peer, "extensions/peer"},
+          {ledger, "extensions/ledger"},
+          {info, "extensions/info"},
+          {trace, "extensions/trace"},
+          {txn, "extensions/txn"},
+          {dkg, "extensions/dkg"},
+          {authorize, "extensions/authorize"},
+          {repair, "extensions/repair"},
+          {print_keys, "extensions/print_keys"}
+         ]},
+        {dev_mode, true},
+        {include_erts, false}
+       ]}.
+
```

Comment:
I see that you hoisted this `relx` out of the `dev` and `aws` profiles to avoid duplication. You kept the `print_keys` extension which is the most recent change to `rebar.config` that I care about for the hotspots.